client/jetpack-onboarding/main.jsx

```diff
@@ -90,6 +93,7 @@ class JetpackOnboardingMain extends React.PureComponent {
 						onBackClick={ this.getNavigationLinkClickHandler( 'back' ) }
 						onForwardClick={ this.getNavigationLinkClickHandler( 'forward' ) }
 						recordJpoEvent={ recordJpoEvent }
+						saveJetpackOnboardingSettings={ saveJetpackOnboardingSettings }
```

Comment:
Did you mean:

```
saveJetpackOnboardingSettings={ this.props.saveJetpackOnboardingSettings }
```

because right now no settings will be saved from any step.

Reply:
Thanks! I fixed that in #22775, but forgot to carry it over here.

Reply:
Fixed in a1ac5a1 -- the nomenclature looks a bit weird there, but it'll make more sense once #22775 lands.

Reply:
Decided to make things a bin nicer: 926eabb